examples/run_bots.sh

```diff
@@ -0,0 +1,13 @@
+export BOT_DEVNET=true
+export BOT_RPC_URL=http://localhost:8545
+export BOT_LOG_FILENAME=testnet_bots
+export BOT_LOG_LEVEL=INFO
+export BOT_MAX_BYTES=100
+export BOT_NUM_LOUIE=0
+export BOT_NUM_FRIDA=0
+export BOT_NUM_RANDOM=4
+export BOT_TRADE_CHANCE=0.1
+export BOT_ALCHEMY=false
+export BOT_ARTIFACTS_URL=http://localhost:80
+
+python examples/evm_bots.py
```

Comment:
Better to put these env variables in front of the python call so that they do not persist after the shell script ends.

Set everything to a variable name at the top of the shell, and then run
`BOT_DEVNET=bot_devnet_var BOT_RPC_URL=bot_rpc_url_var python examples/evm_bots.py` with the others in there as well.